updating/update-using-custom-machine-config-pools.adoc

```diff
@@ -0,0 +1,225 @@
++
+[source,terminal]
+----
+$ oc create -f mcpfoo.yaml
+----
++
+.. Example:
++
+[source,terminal]
+----
+$ oc create -f mcpfoo.yaml
+machineconfigpool.machineconfiguration.openshift.io/mcpfoo created
+----
++
+. To see the list of MCPs present in the cluster and their state
++
+[source,terminal]
+----
+$ oc get machineconfigpool
+----
++
+.. Example:
++
+[source,terminal]
+----
+$ oc get machineconfigpool
+NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+master   rendered-master-b0bb90c4921860f2a5d8a2f8137c1867   True      False      False      3              3                   3                     0                      97m
+mcpbar   rendered-mcpbar-87ba3dec1ad78cb6aecebf7fbb476a36   True      False      False      2              2                   2                     0                      2m18s
+mcpfoo   rendered-mcpfoo-87ba3dec1ad78cb6aecebf7fbb476a36   True      False      False      1              1                   1                     0                      2m42s
+worker   rendered-worker-87ba3dec1ad78cb6aecebf7fbb476a36   True      False      False      0              0                   0                     0                      97m
+----
+
+=== Pause a MCP
+
+Pausing a MCP will keep it from updating to the new OS version by the machine config operator.
+
+[source,terminal]
+----
+$ oc patch mcp/<mcp name> --patch '{"spec":{"paused":true}}' --type=merge
+----
+Example:
+[source,terminal]
+----
+$  oc patch mcp/mcpfoo --patch '{"spec":{"paused":true}}' --type=merge
+machineconfigpool.machineconfiguration.openshift.io/mcpfoo patched
+----
+
+=== Unpause a MCP
+
+Unpausing will enable the nodes in the MCP to move to the new OS version and reboot (if required) it.
+
+[source,terminal]
+----
+$ oc patch mcp/<mcp name> --patch '{"spec":{"paused":false}}' --type=merge
+----
+Example:
+[source,terminal]
+----
+$  oc patch mcp/mcpfoo --patch '{"spec":{"paused":false}}' --type=merge
+machineconfigpool.machineconfiguration.openshift.io/mcpfoo patched
+----
+
+== Steps to remove a node from a MCP
+
+A node must have a role to be properly functioning with in the OpenShift cluster.
+If you want to remove a node from a MCP, you should first relabel the node as a worker as it should be part of worker MCP if is not going to be part of any other MCP. Once it is labeled as worker only then proceed to unlabel it from the MCP.
+
+. Label it as worker if it does not have worker label
++
+[source,terminal]
+----
+$ oc label node <node name>  node-role.kubernetes.io/worker=
+----
++
+. Remove the MCP label
++
+[source,terminal]
+----
+$oc label node <node name> node-role.kubernetes.io/<mcp name>-
+----
++
+. The machine config operator is then going to reconcile the node to the worker pool configuration. Check the output of `oc get mcp` to make sure the worker pool is updated before going to the next step.
++
```

Comment:
@LalatenduMohanty @sdodson @jiajliu 
When I moved the node back to the worker MCP, by removing the label, the `oc get mcp` command showed the worker nodes at the proper machine count (3), but the custom MCP I created stll showed 1. Is this expected? I expected the custom MCP to show 0.

```
$oc get mcp
NAME           CONFIG                                                   UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
master         rendered-master-1203f157d053fd987c7cbd91e3fbc0ed         True      False      False      3              3                   3                     0                      61m
mcp-noupdate   rendered-mcp-noupdate-5ad4791166c468f3a35cd16e734c9028   True      False      False      1              1                   1                     0                      21m
worker         rendered-worker-5ad4791166c468f3a35cd16e734c9028         True      False      False      3              3                   3                     0                      61m
```

Reply:
Yeah, the custom MCP should show 0. But the reconciliation takes few minutes. So if you check the output after few minutes it should be zero.